hack/jenkins/save_build.sh

```diff
@@ -57,24 +58,77 @@ JENKINS_INSTANCE=$(echo $JENKINS_URL | sed -E 's/^\s*.*:\/\///g' | sed -E 's/:.*
 COMP_UID="uid.obj.build.jenkins(instance='$JENKINS_INSTANCE',job_name='$JENKINS_JOB_NAME',build_number='$BUILD_NUMBER')"
 provenance_source_file="$WORKSPACE/provenance/source.json"
 
-# Function to run srp source provenance command
-function source_provenance() {
-    sudo /srp-tools/srp provenance source --scm-type git --name "$PRODUCT_NAME" --path ./ --saveto $provenance_source_file --comp-uid $COMP_UID --build-number ${version_build_num} --version $branch_version --all-ephemeral true --build-type release $@
-}
+# initialize credentials that are required for submission, Credentials value set by jenkins vault plugin
+sudo /srp-tools/srp config auth --client-id=${SRP_CLIENT_ID} --client-secret=${SRP_CLIENT_SECRECT}
+
+# initialize blank provenance in the working directory, $SRP_WORKING_DIR
+sudo /srp-tools/srp provenance init --working-dir $WORKSPACE/provenance
+sudo /srp-tools/srp provenance add-build jenkins --instance $JENKINS_INSTANCE --build-number $BUILD_NUMBER --job-name $JENKINS_JOB_NAME --working-dir $WORKSPACE/provenance
+
+# add an action for the golang build, importing the observations that were captured in the build-golang-app step
+sudo /srp-tools/srp provenance action start --name=amko-build --working-dir $WORKSPACE/provenance
+sudo /srp-tools/srp provenance action import-observation --name=amko-obs --file=$WORKSPACE/provenance/network_provenance.json --working-dir $WORKSPACE/provenance
+sudo /srp-tools/srp provenance action stop --working-dir $WORKSPACE/provenance
+
+# Add an action for make build itself.  Note that it does not actually record the real command line here.  Also note
```

Comment:
please delete commented code.

Reply:
@akshayhavile , these are description which tell what we have to do in the next steps. we need this in the file